packages/parser-manifest/src/schema.json

```diff
@@ -10,13 +10,7 @@
                 },
                 "purpose": {
                     "default": "any",
-                    "enum": [
-                        "any",
-                        "any badge",
-                        "badge",
-                        "badge any",
-                        "maskable"
-                    ],
+                    "pattern": "^\\s*(any|maskable|badge)(\\s+(any|maskable|badge)?)*$",
```

Comment:
https://github.com/webhintio/hint/issues/3791#issuecomment-641935207

We need to add support for `monochrome` as well, I am OK if you think that should be done in a follow-up change.

Reply:
I can add it to this PR